src/containers/Accounts/AccountTransactionTable/AccountTransactionTable.tsx

```diff
@@ -79,7 +79,7 @@ export const AccountTransactionTable = ({
       transactions={transactions}
       loading={loading}
       hasTokensColumn={hasTokensColumn}
-      emptyMessage={emptyMessage && t(emptyMessage)}
+      emptyMessage={emptyMessage && t(emptyMessage as any)}
```

Comment:
Should there be `TODO`s on all these casts to come back and fix later?

Reply:
I dont think so in this case.